### PR TITLE
[VAS-864] feat: add user and groups to bo nav

### PR DESCRIPTION
**Pull request by alessio-cialini**
#### List of Changes

- Introduced users and groups redirection on back office side-menu

#### Motivation and Context

https://pagopa.atlassian.net/browse/VAS-864

#### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as
  expected)

#### Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: src/components/SideMenu/SideMenu.tsx [hunk 1]
```diff
@@ -114,33 +117,53 @@ export default function SideMenu() {
...
+                    {ENV.FEATURES.DASHBOARD.ENABLED && (
+                        <SidenavItem
+                            title={t('sideMenu.users.title')}
+                            handleClick={() => onExit(() => window.location.assign(`${SELFCARE_URL}/users`))}
+                            icon={PeopleAltIcon}
+                            dataTestId={'selfcare-users-test'}
+                        />
+                    )}
+
+                    {ENV.FEATURES.DASHBOARD.ENABLED && (
```
Comment (jacopocarlini) [inline src/components/SideMenu/SideMenu.tsx:156]:
```suggestion
                    { (
```

File: src/components/SideMenu/SideMenu.tsx [hunk 2]
```diff
@@ -114,33 +117,53 @@ export default function SideMenu() {
...
+                            isSelected={
+                                pathname === ROUTES.OPERATION_TABLE_LIST ||
+                                pathname.startsWith(ROUTES.OPERATION_TABLE_LIST)
+                            }
+                            icon={ExtensionIcon}
+                            disabled={isDisabled}
+                            dataTestId="operation-table-test"
+                        />
+                    )}
+
```
Comment (jacopocarlini) [inline src/components/SideMenu/SideMenu.tsx:145]: add a divider

**Review by jacopocarlini — CHANGES_REQUESTED** [review]
add the divider

**Review by svariant — APPROVED** [review]
💪🏻

**Review by jacopocarlini — APPROVED** [review]

**Pull request merged**